pomegranate/base.pyx

```diff
@@ -137,6 +138,29 @@ cdef class Model(object):
 		"""
 		raise NotImplementedError
 
+	def plot( self, n=1000, **kwargs ):
```

Comment:
oops, didn't think of that, I'm plotting some dists for one of my PoCs, and for multivariate dists, overlay plotting of all variables seems satisfactory to me:

![image](https://cloud.githubusercontent.com/assets/15646573/22919661/0040ce44-f2cc-11e6-89b7-7da30f38f1dc.png)

with updated `plot` code like:
```
	sampled_data = numpy.array(self.sample(n))
	for i in range(sampled_data.shape[-1]):
		plt.hist( sampled_data[..., i], **kwargs )
```

Do you think this implementation makes sense?